libopendroneid/opendroneid.h

```diff
@@ -215,34 +225,39 @@ typedef struct __attribute__((__packed__)) {
     uint8_t MessageType : 4;
 
     // Byte 1 [Status][Reserved][NSMult][EWMult] -- must define LSb first
-    uint8_t EWMult: 1;
-    uint8_t NSMult: 1;
-    uint8_t Reserved: 2;
+    uint8_t SpeedMult: 1;
+    uint8_t EWDirection: 1;
+    uint8_t HeightType: 1;
+    uint8_t Reserved: 1;
     uint8_t Status: 4;
 
     // Bytes 2-18
-    uint8_t SpeedNS;
-    uint8_t SpeedEW;
+    uint8_t Direction;
+    uint8_t SpeedHorizontal;
     int8_t SpeedVertical;
     int32_t Latitude;
     int32_t Longitude;
     uint16_t AltitudeBaro;
     uint16_t AltitudeGeo;
-    uint16_t HeightAboveTakeoff;
+    uint16_t Height;
 
     // Byte 19 [VertAccuracy][HorizAccuracy]  -- must define LSb first
     uint8_t HorizAccuracy:4;
     uint8_t VertAccuracy:4;
 
-    // Byte 20 [Reserved2][SpeedAccuracy]  -- must define LSb first
+    // Byte 20 [BaroAccuracy][SpeedAccuracy]  -- must define LSb first
     uint8_t SpeedAccuracy:4;
-    uint8_t TSAccuracy:4;
+    uint8_t BaroAccuracy:4;
 
     // Byte 21-22
     uint16_t TimeStamp;
 
-    // Byte 23-24
-    char Reserved2[2];
+    // Byte 23 [Reserved2][TSAccuracy]  -- must define LSb first
+    uint8_t TSAccuracy:4;
+    uint8_t Reserved2:4;
+
+    // Byte 24
+    char Reserved3[1];
```

Comment:
Why is this an array of size one?

Reply:
Stupid mistake. I will fix it. Thanks for finding.